src/PrestaShopBundle/Translation/CatalogueFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
@thierrymarianne so, this should be a ``TranslationTreeFactory`` instead, right ?